tools/bin/mbedtls-check-missing-changelog

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+help () {
+  cat <<EOF
+Usage: $0 REVISION_RANGE
+Find Mbed TLS pull requests in the specified range of revisions that seem
+to be missing a changelog entry.
+
+This script uses heuristics:
+* Pull requests are merge commits on the first-parent-path in the specified
+  revision range.
+* Pull requests that modify ChangeLog or ChangeLog.d in any way are assumed
+  to be ok.
+* Pull requests whose last commit before merging has an @arm.com author
+  are assumed to be team PR where we put a changelog entry if necessary.
```

Comment:
Nowadays I don't think that's relevant anymore. We do tend to forget changelogs even in team PRs, not significantly less so than in third-party PRs.